pss/outbox/message.go

```diff
@@ -0,0 +1,19 @@
+package outbox
+
+import (
+	"time"
+
+	"github.com/ethersphere/swarm/pss/message"
+)
+
+type outboxMsg struct {
+	msg       *message.Message
+	startedAt time.Time
+}
+
+func NewOutboxMessage(msg *message.Message) *outboxMsg {
```

Comment:
Add comments to all exported declarations. This applies for every declaration in new files in this PR.